ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -3064,17 +3054,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-ui
     workflow: cucushift-agent-vsphere-install-sno
-- as: vsphere-ipi-compact-f14
+- as: vsphere-ipi-compact-etcd-encryption-f14
```

Comment:
since name changed, we may also need to update cron like
`cron: 39 15 10,25 * *`

Reply:
fixed.